packages/legend-query/src/stores/QueryBuilderExplorerState.ts

```diff
@@ -359,6 +360,27 @@ const getQueryBuilderTreeData = (
   return { rootIds, nodes };
 };
 
+export const buildClassPropertyNodes = (
```

Comment:
why do we move this method here instead of leaving it alone in the component? I don't see anything else using it.